src/plugins/python/testmod_python.c

```diff
@@ -0,0 +1,145 @@
+/*
+ * \copydoc python.c
+ */
+
+#include <stdlib.h>
+#include <Python.h>
+
+#ifdef HAVE_KDBCONFIG_H
+#include "kdbconfig.h"
+#endif
+
+#include <tests_plugin.h>
+
+static int Python_AppendToSysPath(const char *path)
+{
+	if (path == NULL)
+		return 0;
+
+	PyGILState_STATE gstate = PyGILState_Ensure();
+	PyObject *sysPath = PySys_GetObject((char *)"path");
+	PyObject *pyPath  = PyUnicode_FromString(path);
+	PyList_Append(sysPath, pyPath);
+	Py_DECREF(pyPath);
+	PyGILState_Release(gstate);
+	return 1;
+}
+
+void init_python()
```

Comment:
static is missing.
The function is part of the workaround explained in CMakeLists.